scripts/inc.updatebinary.sh

```diff
@@ -414,11 +411,18 @@ if [ "$g_conf" ]; then
   sed -e 's|#.*||g' -e 's/\r//g' -e '/^$/d'  "$g_conf_orig" > $g_conf; # Strip user comments from gapps-config
   user_remove_list=$(awk -F "[()]" '{ for (i=2; i<NF; i+=2) print $i }' $g_conf); # Get users list of apk's to remove from gapps-config
   sed -i s/'([^)]*)'/''/g $g_conf; # Remove all instances of user app removals (stuff between parentheses)
+  user_vars_list=$(awk -F "[][]" '{ for (i=2; i<NF; i+=2) print $i }' $g_conf); # Get users vars from gapps-config
```

Comment:
Could we postpostone this user_vars_list then for now? It is powerful, but people who want to change how the script works, will more probably edit the script directly.

What we could do, is to think about better/more configuration style for the longer term, e.g. with more structure and parsing.